ONSdigital/takeon-ui
# Bug/spp 127/ensure unit tests work in concourse

# Description
Fixed broken unit tests, add pytest.ini and .coveragerc files to configure pytest run, which includes ignoring acceptance-tests and test-auth.

JIRA link: https://collaborate2.ons.gov.uk/jira/browse/SPP-127

# How to test?
Run the UI build job in the spp-127 environment and check that the Unit tests task is working.
